java/core/src/main/java/co/worklytics/psoxy/impl/RESTApiSanitizerImpl.java

```diff
@@ -301,6 +303,37 @@ String canonicalizeIp(String ip) {
         }
     }
 
+    MapFunction getRedactExceptPhrases(Transform.RedactExceptPhrases transform) {
+
+        //TODO: alternatively, all different patterns, and preserve ALL matches?
+        // --> means we might enlarge, if the same phrase matches multiple times
+
+        List<Pattern> patterns = transform.getAllowedPhrases().stream()
+            .map(p -> "\\Q" + p + "\\E") // quote it
+            .map(p -> "\\b" + p + "[\\\\s:]*\\b") //boundary match, with optional whitespace or colon at end
```

Comment:
@jlorper this is where should be limiting to word boundaries.

maybe bug is that `"[\\\\s:]*"` is wrong, so it's matching character 's' specifically, instead of `\s` for whitespace as I intended ...

Reply:
yeah, it was that - fixed now and works as expected.